### PR TITLE
feat: LT-1086 - Organisation Flags

### PR DESCRIPTION
**Pull request by cstobart**

### CODE REVIEW

File: organisations/tests/tests_organisation_flags.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+        self.client.put(self.organisation_flag_url, data, **self.gov_headers)
+
+        self.assertEquals(len(self.all_flags), len(self.organisation.flags.all()))
+        for flag in self.all_flags:
+            self.assertTrue(flag in self.organisation.flags.all())
+
+    # TODO: Establish stable activity for organisation avoiding circular imports
+    # def test_flagging_a_organisation_creates_timeline_entries(self):
+    #     """
```
Comment (jfaracik) [inline organisations/tests/tests_organisation_flags.py]: TODO

File: organisations/serializers.py [hunk 2]
```diff
@@ -198,3 +198,13 @@ class Meta:
...
+
+class OrganisationWithFlagsSerializer(OrganisationViewSerializer):
+    flags = serializers.SerializerMethodField()
+
+    def get_flags(self, instance):
+        return list(instance.flags.values('id', 'name'))
+
+    class Meta:
+        model = Organisation
+        fields = '__all__'
```
Comment (BenAndrew-Kainos) [inline organisations/serializers.py]: newline

File: organisations/tests/tests_organisation_flags.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        """
+        data = {
+            'level': 'organisations',
+            'objects': [self.organisation.pk],
+            'flags': [self.team_organisation_flag_1.pk],
+            'note': 'A reason for changing the flags'
+        }
+
+        self.client.put(self.organisation_flag_url, data, **self.gov_headers)
+
```
Comment (jfaracik) [inline organisations/tests/tests_organisation_flags.py]: status code pls

File: organisations/tests/tests_organisation_flags.py [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        data = {
+            'level': 'organisations',
+            'objects': [self.organisation.pk],
+            'flags': [self.other_team_organisation_flag.pk],
+            'note': 'A reason for changing the flags'
+        }
+
+        response = self.client.put(self.organisation_flag_url, data, **self.gov_headers)
+
+        self.assertEquals(0, len(self.organisation.flags.all()))
```
Comment (andytaylorkainos) [inline organisations/tests/tests_organisation_flags.py]: use .`count()` please

File: organisations/tests/tests_organisation_flags.py [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+            'level': 'organisations',
+            'objects': [self.organisation.pk],
+            'flags': [self.good_flag.id],
+            'note': 'A reason for changing the flags'
+        }
+
+        response = self.client.put(self.organisation_flag_url, data, **self.gov_headers)
+
+        self.assertEquals(status.HTTP_400_BAD_REQUEST, response.status_code)
+        self.assertEquals(0, len(self.organisation.flags.all()))
```
Comment (andytaylorkainos) [inline organisations/tests/tests_organisation_flags.py]: same as above

**Pull request merged**